logos/logos-ui/src/app/features/team-detail/tabs/billing/billing-tab.html

```
@@ -195,15 +195,11 @@
         </div>
         <div class="btcol-trend" role="cell" [attr.data-label]="vsLabel()">
           @if (k.trendPct !== null) {
-            <span
-              class="trend-badge"
-              [class.trend-badge--up]="k.trendPct > 0"
-              [class.trend-badge--down]="k.trendPct < 0"
-            >
+            <span class="trend-badge">
               {{ k.trendPct > 0 ? '+' : '' }}{{ k.trendPct.toFixed(1) }}%
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)